docs/docs-content/clusters/edge/local-ui/host-management/edit-user-data.md

```diff
@@ -0,0 +1,113 @@
+---
+sidebar_label: "Edit User Data"
+title: "Edit User Data"
+description: "Instructions for editing user data in Local UI."
+hide_table_of_contents: false
+sidebar_position: 20
+tags: ["edge"]
+---
+
+The **user-data** file is a YAML file that contains installer configuration for the Palette agent. This file is usually
+prepared before the EdgeForge process and is written into the installer ISO in appliance mode deployments. In agent mode
+deployments, the user data is provided directly to the agent mode installation script. For more information about
+EdgeForge, refer to [EdgeForge Workflow](../../edgeforge-workflow/edgeforge-workflow.md). For more information about
+deployment modes, refer to [Deployment Modes](../../../../deployment-modes/deployment-modes.md).
+
+Local UI allows you to make edits to most fields in the **user-data** file after installation has completed before
+cluster creation. All fields exposed in the form in Local UI have corresponding parameters in the installer reference.
+For more information about the fields available in the installer user data, refer to
+[Installer Reference](../../edge-configuration/installer-reference.md).
+
+After an edit has been made, the new settings will apply after the host reboots.
+
+:::preview
+
+:::
+
+## Limitations
+
+- The following fields cannot be edited in Local UI. If you need to use these settings, you must set them correctly
+  during EdgeForge or provide them to the installation script if you are using agent mode.
```

Comment:
```suggestion
- The following fields cannot be edited in Local UI. If you need to update these settings, you must set them correctly
  during EdgeForge or provide them to the installation script if you are using agent mode.
```
Is update the right word here? "use" feels a bit out of place 😅

Reply:
I would take use over update because update implies that settings are already there? Most of these are empty parameters and if users want to use them they have to do it in the user data. I think I can change "settings" to "parameters"